src/pages/first-byte-page/components/Window.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC, ReactNode } from "react";
+import "./Window.css";
+
+interface WindowProps {
+  windowName: string;
+  children: ReactNode;
+}
+
+const Window: FC<WindowProps> = ({ windowName, children }) => {
```

Comment:
I think we should add another prop called "styles" so that the window frame's width and height can grow accordingly,

So when we want a specified width, the overall window can grow accordingly.

Reply:
Kk thats a good idea. I'm not sure if you had a specific way of implementing it in mind but I've made another commit that allows you to specify what width/height the window should be.